shared/src/main/scala/mouse/anyf.scala

```diff
@@ -34,6 +34,15 @@ final class AnyFOps[F[_], A](private val fa: F[A]) extends AnyVal {
   @inline def ||>[G[_]](f: F ~> G): G[A] = f(fa)
   @inline def thrushK[G[_]](f: F ~> G): G[A] = f(fa)
 
+  def asRightIn[L](implicit F: Functor[F]): F[Either[L, A]] =
+    Functor[F].map(fa)(Right(_))
+
+  def asLeftIn[R](implicit F: Functor[F]): F[Either[A, R]] =
+    Functor[F].map(fa)(Left(_))
+
+  def asSomeIn(implicit F: Functor[F]): F[Option[A]] =
```

Comment:
I was the one who added this `In` postfix in the naming within FEither / FOption syntaxes, and I don't think it should be reused any-elsewhere. In the context of the mentioned syntaxes, it emphasizes the intention of applying the provided function to the value of the inner context (Either / Option). But seeing that postfix is used that way is a little bit confusing.

Reply:
Thinking more on this I agree with you. I still feeling that we need this methods to "map and wrap" an effect `F`. 

What about something like `mapAsRight`, `mapAsLeft` and `mapAsSome`. 
At the end they are something like this 
```scala
IO.pure(1).map(_.asRight[String])// IO[Either[String, Int]]

// with the new operator
IO.pure(1).mapAsRight[String]// IO[Either[String, Int]]
```

 I don't know, I don't have many ideas at the moment

Reply:
Hi! I was just checking issues/PRs before proposing exactly the same thing, with the originally proposed names: `asRightIn`/`asLeftIn`/`asSomeIn`.

Maybe you'll find my thoughts on the naming helpful. To me, these operations _are_ performed on the value _in_ the context of an `F`, which is quite similar to `mapIn`, `flatMapIn`. Somehow, `asRightIn`/`asLeftIn`/`asSomeIn` felt natural to me and I actually was a bit surprised to find out they’re missing.

So, any plans of settling on the naming and merging the PR? 😉

Reply:
OK. Lets get this merged, it's sat long enough, and a 2nd user wanting same thing is evidence for usefulness. 

👍 from me, are you ok to merge in current state @danicheg ?